ddtrace/contrib/grpc_aio/aio_client_interceptor.py

```diff
@@ -0,0 +1,238 @@
+import asyncio
+import functools
+from typing import Callable
+from typing import Tuple
+from typing import Union
+
+import grpc
+from grpc import aio
+from grpc.aio._typing import RequestIterableType
+from grpc.aio._typing import RequestType
+from grpc.aio._typing import ResponseIterableType
+from grpc.aio._typing import ResponseType
+
+from .. import trace_utils
+from ... import Pin
+from ... import Span
+from ... import config
+from ...constants import ANALYTICS_SAMPLE_RATE_KEY
+from ...constants import ERROR_MSG
+from ...constants import ERROR_TYPE
+from ...constants import SPAN_MEASURED_KEY
+from ...ext import SpanTypes
+from ...internal.compat import to_unicode
+from ...propagation.http import HTTPPropagator
+from ..grpc import constants
+from ..grpc import utils
+
+
+def create_aio_client_interceptors(pin, host, port):
+    # type: (Pin, str, int) -> Tuple[aio.ClientInterceptor]
```

Comment:
```suggestion
    # type: (Pin, str, int) -> Tuple[aio.ClientInterceptor, ...]
```

I think it should be changed